### PR TITLE
Update CI to use tagging

### PR DESCRIPTION
**Pull request by incompl**
### Test Plan (Steps to test):

1. Make sure test environment built correctly

### Checklist:

- [x] Create a Test Plan
- [x] Changes Communicated (in commits or above)
- [x] Docs Updated
- [x] Storybook Updated

### CODE REVIEW

File: scripts/prebuild.sh [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
-  takeshape branch create --name ${VERCEL_GIT_COMMIT_REF}
-  takeshape import --schema --preserveAuth --branch ${VERCEL_GIT_COMMIT_REF} --from .takeshape/pattern/schema.json
-  echo `takeshape branch versionUrl --name ${VERCEL_GIT_COMMIT_REF}` > .current-branch-url
+
+  # Main branch
+  if [ "$VERCEL_GIT_COMMIT_REF" == "main" ]; then
+    echo `takeshape branch tagVersion --tag ${VERCEL_GIT_COMMIT_SHA}` > .current-branch-url
+
+  # There is a takeshape branch that matches this git branch
+  elif takeshape branch list | grep -q "📁 $VERCEL_GIT_COMMIT_REF (development)"; then
```
Comment (mshick) [inline scripts/prebuild.sh:11]:
```suggestion
  elif takeshape branch url --name ${VERCEL_GIT_COMMIT_REF}; then
```
I think this approach is cleaner. Need this PR to land first though: https://github.com/takeshape/takeshape/pull/3813

**Review by mshick — CHANGES_REQUESTED** [review]
Looks good. Minor suggestions, and merging will be pending the CLI PR landing (should happen soon).

**Comment by incompl** [comment]
> Looks good. Minor suggestions, and merging will be pending the CLI PR landing (should happen soon).

@mshick I have committed your suggestions

**Comment by mshick** [comment]
The CLI is updated, so I think this is good to merge.

**Review by mshick — APPROVED** [review]

**Pull request merged**